### ISSUE TITLE
MicroBit editor in a spin

### ISSUE THREAD

**Issue by andysc** [microsoft/pxt-microbit#2565]
**Describe the bug**
I tried to update a library version in an existing program. When I clicked the update button, the place where the simulator appears showed the spinning circle. 
Since then (about 4 hours ago), when I try to go back into that program from Home, it just shows the microbit logo and "loading..." forever. 
How do I get back to my program code? 

**To Reproduce**
1. go into the blocks editor, click on Javascript view
2. open the Explorer twisty in the left panel
3. find a library that has the update icon next to it
4. click update.
5... wait forever for control to return.

**Expected behavior**
I would expect the update to take a few moments, and then to get back control of the editor.

**Screenshots**

<img width="203" alt="image" src="https://user-images.githubusercontent.com/2693674/71369246-0b231680-25a2-11ea-84a5-9f71d2823329.png">
<img width="297" alt="image" src="https://user-images.githubusercontent.com/2693674/71369305-3b6ab500-25a2-11ea-9eaa-2881862b950c.png">

**Desktop (please complete the following information):**
 - OS: iOS
 - Browser safari
 - Version 13.04

**Comment by abchatra** [comment]
This is probably issue with the extension which is sending the editor to hung state.  Can you try a different browser and let us know?

**Comment by andysc** [comment]
I'm not sure how to get to my MicroBit "home" from another browser - I'm guessing there's a cookie involved?

**Comment by andysc** [comment]
I found a saved link to one of my programmes. Opened that in Firefox (on Mac).
Tried updating the ws2812b extension...
same thing: spinning wheel where the microbit simulation used to be (above the Explorer list).
And if I click "home" from there, and go back into the app, it does the "loading" thing again.
If I do reload on that page, it goes to the editor... but blued-out and still with a spinning wheel.
